extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/i18n/hue_de.properties

```diff
@@ -1 +1,62 @@
+thing-type.hue.0220.label = Farbtemperatur Lampe (wei�)
+thing-type.hue.0220.description = Dimmbare Lampe mit einstellbarer Farbtemperatur.
+
+# thing type configuration
+thing-type.config.hue.bridge.ipAddress.label = IP Adresse
+thing-type.config.hue.bridge.ipAddress.description = Lokale IP Adresse oder Hostname der Hue Bridge.
+thing-type.config.hue.bridge.userName.label = Benutzer
+thing-type.config.hue.bridge.userName.description = Benutzer zur Authentifizierung an der Hue Bridge.
+thing-type.config.hue.bridge.pollingInterval.label = Abfrageintervall
+thing-type.config.hue.bridge.pollingInterval.description = Intervall zur Abfrage der Hue Bridge (in Sekunden).
+thing-type.config.hue.0000.lightId.label = ID der Lampe
+thing-type.config.hue.0000.lightId.description = ID zur Identifikation der Lampe.
+thing-type.config.hue.0010.lightId.label = ID der Steckdose
+thing-type.config.hue.0010.lightId.description = ID zur Identifikation der Steckdose.
+thing-type.config.hue.0100.lightId.label = ID der Lampe
+thing-type.config.hue.0100.lightId.description = ID zur Identifikation der Lampe.
+thing-type.config.hue.0110.lightId.label = ID der Steckdose
+thing-type.config.hue.0110.lightId.description = ID zur Identifikation der Steckdose.
+thing-type.config.hue.0200.lightId.label = ID der Lampe
+thing-type.config.hue.0200.lightId.description = ID zur Identifikation der Lampe.
+thing-type.config.hue.0210.lightId.label = ID der Lampe
+thing-type.config.hue.0210.lightId.description = ID zur Identifikation der Lampe.
+thing-type.config.hue.0220.lightId.label = ID der Lampe
+thing-type.config.hue.0220.lightId.description = ID zur Identifikation der Lampe.
+
+# channel types
+channel-type.hue.color.label = Farbe
+channel-type.hue.color.description = Erm�glicht die Steuerung der Farbe. Erm�glicht ebenfalls die Lampe zu dimmen oder ein- und auszuschalten.
+channel-type.hue.brightness.label = Helligkeit
+channel-type.hue.brightness.description = Erm�glicht die Steuerung der Helligkeit. Erm�glicht ebenfalls die Lampe ein- und auszuschalten.
+channel-type.hue.switch.label = Schalter
+channel-type.hue.switch.description = Erm�glicht das Ein- und Ausschalten.
+channel-type.hue.color_temperature.label = Farbtemperatur
+channel-type.hue.color_temperature.description = Erm�glicht die Steuerung der Farbtemperatur. Von Tageslichtwei� (0) bis Warmwei� (100).
+channel-type.hue.alert.label = Alarm
+channel-type.hue.alert.description = Erm�glicht ein tempor�res Blinken.
+channel-type.hue.alert.state.option.NONE = Kein Blinken
+channel-type.hue.alert.state.option.SELECT = Einmaliges Blinken
+channel-type.hue.alert.state.option.LSELECT = Mehrfaches Blinken
+channel-type.hue.effect.label = Farbeffekt
+channel-type.hue.effect.description = Erm�glicht einen automatischen Farbwechsels.
+
+config-status.error.missing-ip-address-configuration=Es wurde keine IP Adresse f\u00FCr die Hue Bridge angegeben.
```

Comment:
Why don't you use the german Umlaut "ü" here instead of the `\u00FC`? Is there any reason for it?

Except for this minor issue this PR looks fine for me.